apollo-router/src/plugins/telemetry/config_new/logging.rs

```diff
@@ -104,12 +105,36 @@ pub(crate) struct StdOut {
     pub(crate) enabled: bool,
     /// The format to log to stdout.
     pub(crate) format: Format,
+    /// Log rate limiting. The limit is set per type of log message
+    pub(crate) rate_limit: Option<RateLimit>,
```

Comment:
Ideally don't use Option here, you can see if the count is zero instead. Option in config often leads to weirdness that I think is documented in the config guidelines.

Reply:
I added an enabled field in [4318aaa](https://github.com/apollographql/router/pull/4517/commits/4318aaabb2854f672e3911e1b14bae231cc65f2c)